docs/source/tooling/truffle.rst

```diff
@@ -1,59 +1,165 @@
-MythX Security Analysis plugin for Truffle
-==========================================
+.. _tooling.truffle:
 
-MythX Security Analysis plugin for Truffle adds automated smart contract security analysis to the `Truffle framework <https://truffleframework.com>`_.
+Truffle
+=======
 
-Prerequisites
-^^^^^^^^^^^^^
+**MythX for Truffle** is a `Truffle plugin <https://truffleframework.com/docs/truffle/getting-started/writing-external-scripts#third-party-plugin-commands>`_ that adds automated smart contract security analysis to the `Truffle framework <https://truffleframework.com>`_. With this plugin, you can run security analysis directly from your Truffle development environment.
 
-The MythX plugin is compatible with Truffle 5.0 or higher. Note that your Truffle project must compile
+The MythX plugin requires Truffle 5.0 or higher. Note that your Truffle project must compile
 successfully for the security analysis to work.
 
-Quickstart
+Installation
+------------
+
+You can install the plugin on a per-project basis or globally.
+
+.. warning:: 
+
+   **Windows users** have reported potential issues with installation. You may have to install the following dependencies:
+
+   * `Python <https://www.python.org/>`_ (version 3 or higher)
+   * `Windows Build Tools npm package <https://www.npmjs.com/package/windows-build-tools>`_::
+
+       npm install --global --production windows-build-tools
```

Comment:
`windows-build-tools` should automatically install python, and I believe the required version is python 2. It might be good to double check that, but I think we can simplify this to only requiring installing the windows build tools.

We should also mention that an elevated command prompt will be needed for the npm install command, we could perhaps link to the instructions for `node-gyp`, which is our dependency that requires `windows-build-tools`. https://github.com/nodejs/node-gyp#option-1

Reply:
I just installed Truffle Security with Tom, and even though Windows Build Tools reported that it installed Python, we weren't able to get anything working until we ran the standalone Python installer.

Reply:
Actually, I stand corrected. I just tried an install on a clean copy of Windows 10, and indeed, a separate Python install was no necessary.

However, Git was necessary to install truffle-security, so I'll swap out Python for Git